examples/debug-trans-socket/duk_trans_socket_unix.c

```diff
@@ -11,20 +11,20 @@
  * on all platforms.  The default implementation is to use "poll()"
  * but we can switch that by defining "USE_SELECT" within this
  * file.  For example:
- * 
+ *
  * #define USE_SELECT
- * 
+ *
  * If set, instead of leveraging "poll()", this code will leverage
  * "select()".   Discussions on "poll()" vs "select()" can be read
  * about here: https://daniel.haxx.se/docs/poll-vs-select.html
- * 
+ *
  */
 #include <stdio.h>
 #include <string.h>
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <unistd.h>
-#ifndef USE_SELECT
+#if !defined USE_SELECT
```

Comment:
Please use parens here, e.g. `#if !defined(USE_SELECT)`. The current convention is described here: https://github.com/svaarala/duktape/blob/master/doc/code-issues.rst#preprocessor-ifdef-vs-if-defined.